libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-wrapper.component.html

```diff
@@ -8,9 +8,10 @@
 <div
   class="sky-ag-grid"
   [ngClass]="{
```

Comment:
The `ag-theme-` classes are generated when defining a CSS-property-based theme, and need to be present on a parent element.